src/h264-extradata.js

```diff
@@ -0,0 +1,213 @@
+(function() {
```

Comment:
In the future, please try to split refactor PRs from functionality-changing PRs so it's clearer what needs to be reviewed.